.travis.yml

```diff
@@ -17,7 +17,7 @@ notifications:
     on_success: change
     on_failure: always
     on_start: never
-script: npm run lint && npm run cover
+script: npm test
```

Comment:
the coverage is automaticlaly pushed by travis to coveralls, should be `npm run cover` instead to continue updating coverage stats

Reply:
This should still work, it's part of `jest.config.js`

Reply:
You can also see the coverage still being reported here: https://travis-ci.org/glayzzle/php-parser/builds/632443880?utm_source=github_status&utm_medium=notification

Reply:
great 👍